components/server/ee/src/user/stripe-service.ts

```diff
@@ -34,11 +36,15 @@ export class StripeService {
     }
 
     async findCustomerByUserId(userId: string): Promise<Stripe.Customer | undefined> {
-        return this.findCustomerByQuery(`metadata['userId']:'${userId}'`);
+        return this.findCustomerByQuery(
+            `metadata['${ATTRIBUTION_ID_METADATA_KEY}']:'${AttributionId.render({ kind: "user", userId })}'`,
```

Comment:
do you plan to manually update existing customers?

Reply:
They've already been updated. The previous PR changed the subscription queries to set the attribution ID alongside the previous team/user id, this now removes old I'd usage